release-notes/opensearch-index-management-dashboards-plugin.release-notes-2.9.0.0.md

```diff
@@ -0,0 +1,23 @@
+## Version 2.9.0.0 2023-07-11
+
+Compatible with OpenSearch 2.9.0
+
+### Maintenance
+* Increment version to 2.9.0.0. ([#795](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/795))
+* Move index management from plugin section to management section. ([#741](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/741))
+
+### Features
+* Implemented alias action UX. ([#754](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/754))
+
+### Bug Fixes
+* Fix flush flaky test. ([#779](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/779))
+* Fix CI on Node.js v18. ([#785](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/785))
+* Fix: add validation on data.method when using transport.request. ([#801](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/801))
+
+### Refactoring
+* Update release note and minor change to toast message. ([#770](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/770))
+* Minor change for clear cache operation. ([#773](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/773))
+* Wording changes. ([#784](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/784))
```

Comment:
These three were included in 2.8, should we remove them from 2.9 release notes

Reply:
Thank you! Removed.